packages/template-block/src/components/TemplateText.tsx

```diff
@@ -93,17 +94,30 @@ export const TemplateText = ({
         [appBridge, blockId, description, isEditing, saveDescription, templateTextKey],
     );
 
+    const titleClasses = useMemo(() => {
+        const removeTopMargin =
+            '[&>div>*:first-child]:!tw-mt-0 [&>div>[data-slate-editor="true"]>*:first-child]:!tw-mt-0';
+        const removeBottomMargin =
+            '[&>div>*:last-child]:!tw-mb-0 [&>div>[data-slate-editor="true"]>*:last-child]:!tw-mb-0';
```

Comment:
You can't rely on that, do something like here: https://github.com/Frontify/guideline-blocks/blob/59c20b9a0ba18a17d08198892de064bb8bb792b1/packages/callout-block/src/CalloutBlock.tsx#L85

Where we overwrite the theme settings, this you can overwrite the margin-bottom/top of all available text styles, and you don't have to overwrite them via tailwind

Reply:
Thanks for the suggestion! 💪